plasmapy/analysis/time_series/running_moments.py

```diff
@@ -0,0 +1,128 @@
+"""Functionality to calculate running moments of time series"""
+import numpy as np
+
+
+def running_mean(signal, radius):
+    """
+    Calculate the running mean of a sequence
+
+    Parameters
+    ----------
+    signal: |array_like|
+        signal to be averaged.
+
+    radius: int
+        window size is 2*radius+1
+
+    Returns
+    -------
+    |array_like|
+        running mean of signal with length len(signal) - 2*radius
```

Comment:
```suggestion
    Calculate the running mean of a sequence.

    Parameters
    ----------
    signal : 1D |array_like|
        Signal to be averaged.

    radius : int
        The window size is ``2 * radius + 1``.

    Returns
    -------
    |array_like|
        Running mean of signal with length ``len(signal) - 2 * radius``.
```
Only for matching formatting.  The double back ticks will format it as literal text that doesn't get linked.